pkg/lib/authn/authenticator/password/generator.go

```diff
@@ -14,8 +14,9 @@ const (
 	CharListAlphabet     = CharListLowercase + CharListUppercase
 	CharListDigit        = "0123456789"
 	CharListAlphanumeric = CharListAlphabet + CharListDigit
-	// TODO: Define real list of special characters
-	CharListSymbol = "!@#$%^&*()-_=+,.?/:;{}[]`~"
+	// Referenced from "special" character class in Apple's Password Autofill rules.
+	// https://developer.apple.com/documentation/security/password_autofill/customizing_password_autofill_rules
+	CharListSymbol = "-~!@#$%^&*_+=`|(){}[:;\"'<>,.?"
```

Comment:
Missing `]`

Reply:
Fixed up in [45411de](https://github.com/authgear/authgear-server/pull/4535/commits/45411de9813c1ff4836355b4f2bde8f58f65a76e) :bow: